app/models/post.rb

```diff
@@ -1,5 +1,14 @@
 class Post < ApplicationRecord
-  belongs_to :user, foreign_key: 'author_id'
-  has_many :likes, dependent: :destroy
-  has_many :comments, dependent: :destroy
+  belongs_to :author, class_name: 'User'
+  validates :author, presence: true
+  has_many :likes, inverse_of: 'post', dependent: :destroy
+  has_many :comments, inverse_of: 'post', dependent: :destroy
+
+  after_create :update_author_posts_counter
+
+  private
+
+  def update_author_posts_counter
+    author.increment!(:posts_counter)
+  end
 end
```

Comment:
- [ ] Good job working on the Post model but please kindly include a method that returns the 5 most recent comments for a given post.